docs/01-app/03-api-reference/05-config/01-next-config-js/redirects.mdx

```diff
@@ -307,6 +346,8 @@ module.exports = {
 }
 ```
 
+</PagesOnly>
+
 In some rare cases, you might need to assign a custom status code for older HTTP Clients to properly redirect. In these cases, you can use the `statusCode` property instead of the `permanent` property, but not both. To to ensure IE11 compatibility, a `Refresh` header is automatically added for the 308 status code.
```

Comment:
```suggestion
In some rare cases, you might need to assign a custom status code for older HTTP Clients to properly redirect. In these cases, you can use the `statusCode` property instead of the `permanent` property, but not both. To ensure IE11 compatibility, a `Refresh` header is automatically added for the 308 status code.
```